runtime/doc/lsp.txt

```diff
@@ -81,13 +81,14 @@ the options are empty or were set by the builtin runtime (ftplugin) files. The
 options are not restored when the LSP client is stopped or detached.
 
 GLOBAL DEFAULTS
-                                *grr* *gra* *grn* *gri* *grt* *i_CTRL-S* *an* *in*
+                        *grr* *gra* *grn* *gri* *grt* *grc* *i_CTRL-S* *an* *in*
 These GLOBAL keymaps are created unconditionally when Nvim starts:
 - "grn" is mapped in Normal mode to |vim.lsp.buf.rename()|
 - "gra" is mapped in Normal and Visual mode to |vim.lsp.buf.code_action()|
 - "grr" is mapped in Normal mode to |vim.lsp.buf.references()|
 - "gri" is mapped in Normal mode to |vim.lsp.buf.implementation()|
 - "grt" is mapped in Normal mode to |vim.lsp.buf.type_definition()|
+- "grc" is mapped in Normal and Visual mode to |vim.lsp.document_color.color_presentation()|
```

Comment:
This seems like a bit of an obscure feature to dedicate a default mapping to. In comparison to the other default mappings, which are valid with most (all?) language servers, this one only works with a select few (as a data point, I’ve never used a language server that supports document colors).

Reply:
That's fair. I'll drop the commit.